implementing-nosto/advanced-implementation/accessing-state-with-selectors.md

```diff
@@ -0,0 +1,59 @@
+# Accessing state with selectors
+
+`useAppState` hook provides you full access to the entire application state.
+
+{% code title="State example" %}
+```jsx
+import { useAppState } from '@nosto/preact'
+
+export default () => {
+    const state = useAppState()
```

Comment:
sorry, I should have raised this earlier
useAppState is deprecated, I wouldn't document usage at all, but just promote useAppStateSelector

Reply:
Will be done in a follow-up